### ISSUE TITLE
Release of PyTorch pretrained model and ablation

### ISSUE THREAD

**Issue by d-li14** [iamhankai/ghostnet.pytorch#17]
Thanks for your awesome work. I have provided the PyTorch pre-trained GhostNet 1.0x with comparable accuracy in https://github.com/d-li14/ghostnet.pytorch. Furthermore, I conducted ablation experiments about different training settings, which may echo some questions in #3, #8.

**Comment by iamhankai** [comment]
Great job! Thanks for your works.
In addition, 360 epochs, warmup and lower dropout ratio may help to further improve the performance.

**Comment by d-li14** [comment]
Thanks for your kind suggestion! I have tried training longer until 360 epochs, but it brings about marginal benefit. I have also applied warmup in the original experiments. Seen from the training curve, I conjecture that keeping the dropout ratio as 0.2 will be a better choice compared to 0.1, 0.15. Correct me if I am wrong.

**Comment by iamhankai** [comment]
Official pytorch pretrained model is released: https://github.com/huawei-noah/ghostnet/tree/master/pytorch
